src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -192,6 +192,12 @@ Copyright (c) .NET Foundation. All rights reserved.
         <RelativePath>@(_DebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
 
+      <!-- Copy the documentation file (.xml), if any -->
+      <ResolvedFileToPublish Include="@(_DebugSymbolsIntermediatePath)"
```

Comment:
There is a good use case for using documentation files at runtime in web projects. I've used them to show programatically-generated API docs in a very similar way to what @masterjs and @dasMulli described in the issue. I'm not sure if the same use case exists for NuGet packages. That seems like it would much less common. P2P references probably fall somewhere in between.